ush/parsing_namelists_FV3.sh

```diff
@@ -216,6 +224,13 @@ if [ $CCPP_SUITE = "FV3_GSD_v0" ]; then
   bl_mynn_tkeadvect=${bl_mynn_tkeadvect:-".true."}	! In config.fcst
   bl_mynn_edmf_mom=${bl_mynn_edmf_mom:-"1"}	! In config.fcst
 EOF
+elif [ $CCPP_SUITE = "S2S" ]; then
```

Comment:
There is not a CCPP_SUITE at this time defined as "S2S" this is a duplication of the if block above for the CCPP_SUITE that the s2s model is using right now: FV3_GFS_v15p2_coupled

Reply:
I will fix it. @JessicaMeixner-NOAA